packages/shared/src/components/search/SearchSourceList.tsx

```diff
@@ -29,6 +29,10 @@ export const SearchSourceList = ({
     limit: 3,
   });
 
+  if (!isLoading && sources?.length === 0) {
```

Comment:
I think the condition should have been `!sources?.length`. Undefined or null would be false on this condition.

Reply:
Nvm. I see the other PR now with the right condition.